zebra-chain/src/primitives/zcash_history.rs

```diff
@@ -261,21 +263,74 @@ fn block_to_history_node(
     // There is no direct `std::primitive::u128` to `bigint::U256` conversion
     let work = bigint::U256::from_big_endian(&work.as_u128().to_be_bytes());
 
-    let sapling_tx_count = count_sapling_transactions(block);
+    let sapling_tx_count = count_sapling_transactions(block.clone());
```

Comment:
🤔

Reply:
This feels like it could be a method on `Block`?
It probably doesn't matter much either way.